pandas/core/arrays/base.py

```diff
@@ -2256,6 +2256,19 @@ def _mode(self, dropna: bool = True) -> Self:
         # ndarray[Any, Any]]", expected "Self")
         return mode(self, dropna=dropna)  # type: ignore[return-value]
 
+    def round(self, decimals: int = 0, *args, **kwargs) -> Self:
+        # Implementer note: This is a non-optimized default implementation.
+        # Implementers are encouraged to override this method to avoid
+        # elementwise rounding.
+        if not self.dtype._is_numeric or self.dtype._is_boolean:
+            raise TypeError(
+                f"Cannot round {type(self)} dtype as it is non-numeric or boolean"
+            )
+        return self._from_sequence(
+            [round(element) if not isna(element) else element for element in self],
```

Comment:
`self.isna` or the `self.dtype.na_value` should probably be used to check for na instead of `isna`